### DISCUSSION TITLE
Login with Passkey all the sudden ask for Master password! Did something changed?

### DISCUSSION THREAD

**Discussion by BobWs** [dani-garcia/vaultwarden#4690] [General]
Hi,
Since a few days when I try to login on supported sites where I have setup Passkey, the Bitwarden Firefox Extension keep asking me for the master password to unlock Passkey, even if the extension is already unlocked.

![Screen_Shot](https://github.com/dani-garcia/vaultwarden/assets/5485820/2edba574-3d6b-4a9f-a50b-51712b2e9c26)

Before when using the saved passkeys I only had to unlock the extension once and I could login.

Is something changed with the Bitwarden extension? Or is this the default behavior and was I facing a bug earlier.
This way it is very annoying to use Passkey.

**Comment by ilopmar** [comment]
I though I was going crazy. The same started to happen to my a couple of weeks ago and I agree with you it's annoying. I'm using Brave and the behaviour is the same.

**Reply by BobWs** [reply]
Yes it’s on all the browsers I have tested (Firefox, Safari, Duckduckgo), so it must be due some changes in the extension. Very annoying!

**Reply by tmeuze** [reply]
I was wondering if Bitwarden is now respecting the [FIDO2 PIN requirement](https://support.yubico.com/hc/en-us/articles/4402836718866-Understanding-YubiKey-PINs) to some degree - but I just tried setting up a passkey for AWS (only service I know of that doesn't request my FIDO PIN), and Bitwarden did require I unlock.

**Reply by ilopmar** [reply]
It seems it's "working" again, at least I can now confirm the passkey without my master password.

**Reply by BobWs** [reply]
Looks that way indeed, I could login here on Github with my passkey without the popup for my master password!
